earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +51,12 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+fusiontestdir = os.path.join(testdir, 'fusion')
+shutil.rmtree(fusiontestdir, ignore_errors=True)
+shutil.copytree('../../fusion', fusiontestdir)
```

Comment:
I'm a little torn on this issue. On the one hand, it seems more natural to run the tests from the build directory instead of src, and I certainly don't want to keep track of a bunch of random temp files in .gitignore. But, on the other, copying all the test data can end up taking up a lot of space. I think ideally we would read the data out of the source data but keep the working directory under <builddir>/bin/tests. I'll leave this up to you, @bradh, how much more effort you'd like to put into it. But, I do agree with @andreisel that we should minimize the data that is copied by limiting to the testdata directory if possible. It's 113MB for testdata vs. 317M for all of fusion.